.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 public/build/
+yarn.lock
```

Comment:
Hm - denne burde vi vel egentlig legge til?

Da repoet ble satt opp fantes ikke yarn, men gir vel mening å bruke det nå. Evt. `package-lock.json` fra nyere NPM, men Yarn er fine for min del i hvert fall